Bump graphene and graphene-django versions

New versions of those libraries are out:
- https://github.com/graphql-python/graphene/releases/tag/v2.1.3
- https://github.com/graphql-python/graphene-django/releases/tag/v2.1.0

I tried to quickly do the upgrade but the following error occurs (I haven't yet investigated it):
```
tests/api/test_order.py:10: in <module>
    from saleor.graphql.order.mutations.draft_orders import (
saleor/graphql/order/mutations/draft_orders.py:12: in <module>
    from saleor.graphql.order.types import Order
saleor/graphql/order/types.py:9: in <module>
    class Order(CountableDjangoObjectType):
../../.pyenv/versions/3.7.0/envs/saleor3.7/lib/python3.7/site-packages/graphene/utils/subclass_with_meta.py:52: in __init_subclass__
    super_class.__init_subclass_with_meta__(**options)
saleor/graphql/core/types.py:48: in __init_subclass_with_meta__
    *args, connection=countable_conn, **kwargs)
../../.pyenv/versions/3.7.0/envs/saleor3.7/lib/python3.7/site-packages/graphene_django/types.py:76: in __init_subclass_with_meta__
    construct_fields(model, registry, only_fields, exclude_fields), _as=Field
../../.pyenv/versions/3.7.0/envs/saleor3.7/lib/python3.7/site-packages/graphene_django/types.py:29: in construct_fields
    converted = convert_django_field_with_choices(field, registry)
../../.pyenv/versions/3.7.0/envs/saleor3.7/lib/python3.7/site-packages/graphene_django/converter.py:57: in convert_django_field_with_choices
    converted = registry.get_converted_field(field)
../../.pyenv/versions/3.7.0/envs/saleor3.7/lib/python3.7/site-packages/graphene_django/registry.py:28: in get_converted_field
    return self._field_registry.get(field)
E   TypeError: unhashable type: 'TaxedMoneyField'
```

--- a/requirements.in
+++ b/requirements.in
@@ -33,7 +33,7 @@ faker>=0.7.7
 freezegun>=0.3.9
 google-measurement-protocol>=1.0.0
 google-i18n-address>=2.3.0
-graphene-django>=2.0.dev
+graphene-django>=2.1
 jsonfield>=2.0.2
 Markdown>=2.4
 maxminddb

--- a/requirements.in
+++ b/requirements.in
@@ -16,7 +16,7 @@ django-impersonate==1.3
 django-mptt>=0.7.1
 django-payments>=0.12.1
 django-phonenumber-field>=1.3.0
-django-prices>=1.0.1
+django-prices>=1.0.2
 django-prices-openexchangerates>=1.0.0
 django-prices-vatlayer>=1.0.1
 django-redis>=4.0.0

--- a/requirements.txt
+++ b/requirements.txt
@@ -5,6 +5,7 @@
 #    pip-compile --output-file requirements.txt requirements.in
 #
 amqp==2.2.2               # via kombu
+aniso8601==3.0.2          # via graphene
 asn1crypto==0.24.0        # via cryptography
 attrs==17.4.0             # via pytest
 babel==2.5.3
@@ -59,9 +60,9 @@ faker==0.8.11
 freezegun==0.3.10
 google-i18n-address==2.3.0
 google-measurement-protocol==1.0.0
-graphene-django==2.0.0
-graphene==2.0.1           # via django-graphql-jwt, graphene-django
-graphql-core==2.0         # via graphene, graphql-relay
+graphene-django==2.1.0
+graphene==2.1.3           # via django-graphql-jwt, graphene-django
+graphql-core==2.1         # via graphene, graphene-django, graphql-relay
 graphql-relay==0.4.5      # via graphene
 html5lib==1.0.1           # via bleach, weasyprint
 idna==2.6                 # via cryptography, requests

--- a/requirements.txt
+++ b/requirements.txt
@@ -44,7 +44,7 @@ django-payments==0.12.3
 django-phonenumber-field==2.0.0
 django-prices-openexchangerates==1.0.0
 django-prices-vatlayer==1.0.1
-django-prices==1.0.1
+django-prices==1.0.2
 django-recaptcha==1.4.0
 django-redis==4.8.0
 django-render-block==0.5  # via django-templated-email

--- a/saleor/graphql/core/fields.py
+++ b/saleor/graphql/core/fields.py
@@ -6,10 +6,10 @@ from .types import Money, TaxedMoney
 
 
 @convert_django_field.register(TaxedMoneyField)
-def convert_field_tax(field, registry=None):
+def convert_field_taxed_money(field, registry=None):
     return graphene.Field(TaxedMoney)
 
 
 @convert_django_field.register(MoneyField)
-def convert_field_tax(field, registry=None):
+def convert_field_money(field, registry=None):
     return graphene.Field(Money)

--- a/saleor/graphql/order/types.py
+++ b/saleor/graphql/order/types.py
@@ -18,8 +18,6 @@ class Order(CountableDjangoObjectType):
         description='User-friendly order status.')
     captured_amount = graphene.Field(
         Money, description='Amount captured by payment.')
-    total_price = graphene.Field(
-        TaxedMoney, description='Total price of the order.')
 
     class Meta:
         description = 'Represents an order in the shop.'
@@ -49,11 +47,6 @@ class Order(CountableDjangoObjectType):
     def resolve_status_display(self, info):
         return self.get_status_display()
 
-    def resolve_total_prize(self, info):
-        payment = self.get_last_payment()
-        if payment:
-            return payment.get_total_price()
-
 
 class OrderHistoryEntry(CountableDjangoObjectType):
     class Meta:
@@ -69,7 +62,7 @@ class OrderLine(CountableDjangoObjectType):
         model = models.OrderLine
         interfaces = [relay.Node]
         exclude_fields = [
-            'variant', 'unit_price_gross', 'unit_price_net', 'order']
+            'order', 'unit_price_gross', 'unit_price_net', 'variant']
 
 
 class OrderNote(CountableDjangoObjectType):

--- a/saleor/graphql/product/fields.py
+++ b/saleor/graphql/product/fields.py
@@ -1,6 +1,6 @@
 import graphene
 from django import forms
-from graphene_django.form_converter import convert_form_field
+from graphene_django.forms.converter import convert_form_field
 
 from .scalars import AttributeScalar
 

--- a/saleor/graphql/product/resolvers.py
+++ b/saleor/graphql/product/resolvers.py
@@ -49,7 +49,7 @@ def resolve_collections(info, query):
 def resolve_products(info, category_id, query):
     user = info.context.user
     queryset = products_visible_to_user(
-        user=user).prefetch_related('Category').distinct()
+        user=user).prefetch_related('category').distinct()
     queryset = filter_by_query_param(queryset, query, PRODUCT_SEARCH_FIELDS)
     if category_id is not None:
         category = get_node(info, category_id, only_type=Category)

--- a/tests/api/test_order.py
+++ b/tests/api/test_order.py
@@ -46,12 +46,11 @@ def test_order_query(admin_api_client, fulfilled_order):
                                 fulfillmentOrder
                             }
                         }
-                        
                     }
                     history {
                         totalCount
                     }
-                    totalPrice {
+                    total {
                         net {
                             amount
                         }
@@ -146,7 +145,7 @@ def test_draft_order_create(
                         voucher {
                             code
                         }
-                        
+
                     }
                 }
         }

